scripts/slave_setting/sshfs/resync_shared_dirs.sh

```diff
@@ -10,8 +10,17 @@ dirs=("repository" "repo" "LOGS")
 echo "List of sshfs mounts:"
 mount | grep sshfs
 
+
 for dir_to_resync in "${dirs[@]}"
 do
+
```

Comment:
Please change LOGS to ${dir_to_resync} in lines 17-21 because we should check all mounted directories not only a LOGS.